.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -17,6 +17,16 @@ body:
       description: >
         Shown in the title bar, logs, and long press on the game icon.  If it's outside a game, put 'None.'
 
+ type: input
+    id: Patched_Iso
+    validations:
+      required: true
+    attributes:
+      label: Game or games this happens in
```

Comment:
The label seems off?  Maybe it should be "Using a patched ISO?"

I feel like people won't often correctly fill this in anyway.... I guess we can ask.

-[Unknown]